crates/test_utils/wasm/wasm_workspace/migrate_new/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "test_wasm_migrate_new"
+description = "Test wasm for DNA migration, intended to be the migration target"
+version = "0.0.1"
+authors = [ "thedavidmeister", "thedavidmeister@gmail.com" ]
```

Comment:
He keeps writing stuff even in his absence!

Reply:
Haha, the dangers of copy paste to get the features right for these crates - tidied up :)